src/you_get/extractors/bilibili.py

```diff
@@ -403,6 +435,8 @@ def bilibili_download_playlist_by_url(url, **kwargs):
     elif 'favlist' in url:
         # this a fav list folder
         download_video_from_favlist(url, **kwargs)
+    elif 'video' in url:
```

Comment:
You may want to restrict this to domain space.bilibili.com (or alike)

Otherwise when using `-l` to download a regular video (https://www.bilibili.com/video/....) it will be captured by `download_video_from_totallist` and raises an error.